docs/user/openstack/deploy_baremetal_workers.md

```diff
@@ -0,0 +1,188 @@
+* [Ironic bare metal service][2] can listen for and PXE-boot nodes in tenant networks
+
+## Conciderations whe deploying bare-metal workers
+
+* Long boot times for bare-metal machines may result in deployment timeout
+
+    The first time a bare-metal server is booted the boot time could be significantly longer that
+    the first time a vm server is booted. This longer boot time could exceed the timeout settings of 
+    the installer. If that occurs, the deployment will fail with a timeout error.  The deployment maybe
+    restarted and completed by re-running  the installer with the appropriate *wait-for*
+    command. For example:
+        
+        ./openshift-install wait-for bootstrap-complete --log-level debug 
+          
+
+## Deploying cluster with BM workers on tenant network deployed by the installer
+The initial cluster is deployed using bare-metal workers only. Bare-metal workers and control-plane vms are all
+attached to the tenant network provisioned by the installer.
+
+- Requirements: 
+    - Cloud provideer is configure to allow creation of  OpenStack subnets that supports attaching vms and baremental servers to it. 
+
+- Create install-config.yaml:
+
+    - Set compute.[worker].platform.openstack.type to the baremetal server flavor.
+    - Set control-plane.platform.openstack.type to the vm flavor which will be used by the control-plane masters.
+    - Set platform.openstack.computeFlavor to the vm flavor which will be used by the control-plane masters.
+
+        For example:
+ 
+                controlPlane:
+                   platform:
+                     openstack:
+                       type: <vmComputeFlavorForMasters>
+                 
+                 ... other settings
+                 
+                 compute:
+                 - architecture: amd64
+                   hyperthreading: Enabled
+                   name: worker
+                   platform:
+                     openstack:
+                       type: <baremetalComputeFlavor>
+                   replicas: 3
+         
+                 ... other settings
+                 
+                 platform:
+                   openstack:
+                     computeFlavor: <vmComputeFlavorForMasters>
+
+
+- Run the openshift installer:
+
+      ./openshift-install create cluster --log-level debug
+        
+- Wait for installer to complete. 
+
+    If the installer times out waiting for the baremetal workers to complete booting,
+    restart the installation usin the appropriate *wait-for* command:
+    
+        ./openshit-install watier-for bootstrap-complete --log-level debug
+
+## Deploying cluster with BM workers on preexisting network
+Initial cluster is deployed using bare-metal workers only. Bare-metal workers are attached to
+a preexisting network. 
+
+- Requirements: 
+    - An OpenStack subnet which supports attaching vms and baremental servers to is preprovisioned by the user. 
```

Comment:
Maybe here we can use the phrasing "provider network"

Reply:
provider network is a very specific type of network in openstack.  Is that the only type of network we will support?

Reply:
ACK, but I find "preprovisioned by the user" too specific in the other direction: the network may be "provisioned" by the cloud admin, or am I missing something

Reply:
lets drop "by the user"